package.json

```diff
@@ -52,6 +52,7 @@
     "@mean-dao/msp": "3.0.1-alpha.13",
     "@metaplex-foundation/js": "0.17.5",
     "@metaplex-foundation/mpl-token-metadata": "~2.5.1",
+    "@mithraic-labs/psy-american": "0.2.3",
```

Comment:
We are trying to minimise the dependencies. Would it be possible to implement this without this dependency?